scrc/utils/overview_creation.py

```diff
@@ -0,0 +1,47 @@
+from root import ROOT_DIR
+import os
+import sys
+import csv
+from tqdm import tqdm
+from scrc.utils.court_names import get_all_courts
+
+csv.field_size_limit(sys.maxsize)
+
+
+#   creates an overview of the size of the generated datasets and exports them in a csv file
+def create_overview(include_all=True, path="data/datasets", export_path="data", export_name="overview.csv"):
```

Comment:
please use the paths from the config. Otherwise it will break if they are changed.

Reply:
Makes sense, done.